src/tests/fixtures/resolver_context/ContextValueTypedAsNever.ts.expected

```diff
@@ -0,0 +1,18 @@
+-----------------
+INPUT
+----------------- 
+/** @gqlType */
+export class Query {
+  /** @gqlField */
+  greeting(args: never, ctx: never): string {
+    return ctx.greeting;
+  }
+}
+
+-----------------
+OUTPUT
+-----------------
+src/tests/fixtures/resolver_context/ContextValueTypedAsNever.ts:4:30 - error: Expected context parameter's type to be a type reference Grats validates that your context parameter is type-safe by checking all context values reference the same type declaration.
+
+4   greeting(args: never, ctx: never): string {
```

Comment:
Is it not possible to allow `unknown` as the type for `ctx`?

Reply:
We could, just haven't added it yet! Unclear if it's needed though.  I guess if you need to access the `info` argument after the context value and you don't actually ever use the context value?

Reply:
Yeah, that's one use case. It would be a just escape hatch, in case we need to add the context argument (for whatever reason) but we don't want to deal with typing the context _right_ now.

However, if it's easy to add, we could not add it now, and wait for (less theoretical, more practical) reasons to add it and make a better decision then.